.circleci/config.yml

```diff
@@ -6,13 +6,14 @@ jobs:
   build:
     docker:
       # specify the version
-      - image: circleci/golang:1.8
+      - image: fortio/fortio.build:v4
```

Comment:
fortio now has its own build image? :stuck_out_tongue_closed_eyes: 

curious: how will we track golang versioning in this image?

Reply:
the linters were being a pain so yes I made a build image
it's in the repo, it's currently using go1.8.5 (I have an issue open to evaluate switching to 1.9)